Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/box.yml

```diff
@@ -368,3 +368,164 @@
     contents:
     - id: FoodDonkpocketDink
       amount: 6
+
+- type: entity
+  id: HappyHonkBase
```

Comment:
Base should be the prefix not the suffix, also why isn't happyhonk combined into this?

Reply:
i changed this so the base is now clown box reducing the number by 2 entities, but i added a new entity to be used in the random food spawner which i  added it to (includes toy, burger, drink). Hope this is better than previously.